### ISSUE TITLE
How to disable warning when opening multiple tabs.

### ISSUE THREAD

**Issue by jtheletter** [brave/brave-browser#28020]
I have a bookmark folder of multiple tabs that I open daily. Every time, Brave browser asks, “Are you sure you want to open 15 tabs?” My answer is always yes. How do I stop this from asking?

Version 1.47.171 Chromium: 109.0.5414.87 (Official Build) (arm64)

**Comment by rebron** [comment]
<img width="488" alt="Screenshot 2023-01-27 at 10 57 46 AM" src="https://user-images.githubusercontent.com/5951041/215171810-da736e8e-765f-4788-925f-a193d5a4210f.png">

**Comment by share3141** [comment]
This is a pain. Please fix or allow us an option to turn this off.

**Comment by jamincollins** [comment]
I have a similar folder that I frequently open all the tabs in.  It would be great if there was an option to disable this prompt on a per folder basis.

**Comment by Pellatonian** [comment]
Just moving to Brave and this is my biggest bugbear. Either let me switch it off or at least set my own limit of the number of messages which will trigger the alert.

**Comment by russoj88** [comment]
Any update on this?

My use case is every morning I open about 15 tabs to check on things.

**Comment by leodom** [comment]
Is the 15 hardcoded somewhere ? any Advanced option to change it ?

**Comment by ourichermath** [comment]
I don't see any option to modify this in brave://flags .  The issue got added to "Front End" back in May.  Any update?

**Comment by Fyne5** [comment]
Yes, me too. Now 2025 and RAM is no problem. Just open the world.

**Comment by vdb86** [comment]
Can we please have an option to disable this warning?

**Comment by jtheletter** [comment]
Three years later, issue persists.
